packages/renderer/src/lib/preferences/PreferencesProviderInstallationModal.svelte

```diff
@@ -8,7 +8,7 @@ import type { CheckStatus, ProviderInfo } from '/@api/provider-info';
 import IconImage from '../appearance/IconImage.svelte';
 
 interface Props {
-  providerToBeInstalled: { provider: ProviderInfo; displayName: string };
+  providerToBeInstalled: { provider: ProviderInfo; displayName: string } | undefined;
```

Comment:
This seem to be always defined when called in renderer.

<img width="867" height="174" alt="Screenshot 2025-09-10 at 16 01 38" src="https://github.com/user-attachments/assets/79aa3972-2474-49ff-b20d-04d788e12fa5" />


I would not change the interface of the component, but I would update the test